docs/Earn/liquid-staking/bnb/stake-bnb.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Stake BNB On ankr.com
+title: Stake BNB on ankr.com
 id: stake-bnb
 ---
 
-# Stake BNB On ankr.com
+# Stake BNB on ankr.com
```

Comment:
remove 'on ankr.com' as it is in Ankr Earn.

Reply:
thanks, done.